utils/build-script-impl

```diff
@@ -805,15 +814,20 @@ function is_cross_tools_deployment_target() {
 # A list of deployment targets that we compile or cross-compile the
 # Swift standard library for.
 STDLIB_DEPLOYMENT_TARGETS=()
-
-case "$(uname -s)" in
-    Linux)
+# FIXME: This will not work while trying to cross-compile
+# It gets the architecture from the system processor arch.
```

Comment:
I think this one can be removed as well.

I hope I am understanding this correctly. Would be good if someone else could confirm my logic:
`STDLIB_DEPLOYMENT_TARGETS` specifies all targets that the native build machine is capable of building.
See how `Darwin \x86_64` contains the `iphoneos-arm` targets?
This means, that a `Darwin\ x86_64` build machine is able to (cross-)compile for all of the listed targets.